docs/Installation/Installation_Guide.rst

```diff
@@ -1,579 +1,581 @@
-
-   ::
-
-    $ kubectl get pods -n default -o wide
-
-   This displays a view of pod creation. For a few minutes after the Helm
-   install, some pods will show CrashLoopBackOff issues. This is expected
-   behavior, because there is no launch order between pods. After a few
-   minutes, all pods will enter Running mode.
-
-5. To register your Zenko instance to Orbit, get your Cloudserver’s name:
-
-   ::
-
-    $ kubectl get -n default pods | grep cloudserver
-    my-zenko-cloudserver-76f657695-j25wq              1/1   Running   0       3m
-    my-zenko-cloudserver-manager-c76d6f96f-qrb9d      1/1   Running   0       3m
-
-   Then grab your Cloudserver’s logs with the command:
-
-   ::
-
-     $ kubectl logs my-zenko-cloudserver-<id> | grep 'Instance ID'
-
-
-   Using the present sample values, this command returns:
-
-   ::
-
-     $ kubectl logs my-zenko-cloudserver-76f657695-j25wq | grep 'Instance ID'
-
-     {"name":"S3","time":1532632170292,"req_id":"effb63b7e94aa902711d",\
-     "level":"info","message":"this deployment's Instance ID is \
-     7586e994-01f3-4b41-b223-beb4bcf6fff6","hostname":"my-zenko-cloudserver-\
-     76f657695-j25wq","pid":19}
-
-   Copy the instance ID.
-
-6. Open https://admin.zenko.io/user in a web browser. You may be prompted to
-   authenticate through Google.
-
-7. Click the button to **Register My Instance**.
-
-8. Paste the instance ID into the Instance ID dialog. Name the instance what
-   you will.
-
-Your instance is registered.
+.. spelling::
+
+   Quickstart
+   subdirectory
+   Todo
+
+Installation Guide
+++++++++++++++++++
+
+While it is possible to run Zenko on a single machine, it's designed for
+cluster operation. If you can set up a Kubernetes cluster on your own, review
+the **General Cluster Requirements** and skip to **Installing Zenko**, below.
+
+Otherwise, you can set up a cluster quickly using MetalK8s_, Scality's
+open-source Kubernetes cluster project, as described in "Setting Up a Metal K8s
+Cluster," below.
+
+The following section describes general cluster requirements, which are tested
+on Metal K8s. Because MetalK8s is designed to operate without support from
+public cloud resources, the following sizing requirements are assumed good for
+other cloud Kubernetes deployments, where such resources are preinstalled and
+available on demand.
+
+####################
+Setting Up a Cluster
+####################
+
+****************************
+General Cluster Requirements
+****************************
+
+Setting up a testing cluster requires at least three machines (these can be
+VMs) running CentOS_ 7.4 (The recommended mimimum for Zenko production service
+is five server nodes with three masters/etcds, but for testing and
+familiarization, three masters and three nodes is fine). You must have SSH
+access to these machines and they must have SSH access to each other. (You
+can copy SSH credentials from one machine to the next and log in once to

```

Comment:
Never copy SSH private keys to other machines... There's no need to do it when deploying MetalK8s, and it's bad practice. Worst case you use agent forwarding.